src/views/ContactmomentView.vue

```diff
@@ -1,82 +1,147 @@
+            <li>
+              <button
+                @click="showKlantenSearch = true"
+                class="utrecht-button utrecht-button--secondary-action"
+              >
+                Klanten zoeken
+              </button>
+            </li>
+          </menu>
+          <klant-details
+            v-if="contactmomentStore.klant"
+            :klant="contactmomentStore.klant"
+          />
+          <contactmomenten-overzicht v-if="klantId" :klant-id="klantId" />
+        </article>
+      </template>
+
+      <!-- ZAKEN -->
+      <template #[TabsContactmoment.zaken]>
+        <article>
+          <utrecht-heading :level="1" model-value class="zaak-title">
+            Zaken
+          </utrecht-heading>
+          <tabs-component v-model="activeTab" class="zaak-tabs">
+            <template #[Tabs.personenZoeker]>
+              <persoon-zoeker @zakenZoeken="onZakenZoeken" />
+            </template>
+            <template #[Tabs.zakenZoeker]>
+              <zaak-zoeker :populatedBsn="curentBsn" />
+            </template>
+          </tabs-component>
+        </article>
+      </template>
+    </tabs-component>
   </main>
   <contactmoment-starter />
 </template>
 
 <script setup lang="ts">
-import ZaakZoeker from "@/features/zaaksysteem/ZaakZoeker.vue";
 import PersoonZoeker from "@/features/brp/PersoonZoeker.vue";
-import { ContactmomentStarter } from "@/features/contactmoment";
+import {
+  ContactmomentStarter,
+  ContactmomentenOverzicht,
+} from "@/features/contactmoment";
 import { UtrechtHeading } from "@utrecht/web-component-library-vue";
-import { ref } from "vue";
+import { ref, computed } from "vue";
+import { KlantZoeker, KlantDetails } from "@/features/klant";
+import { useContactmomentStore, type Klant } from "@/stores/contactmoment";
+import TabsComponent from "@/components/TabsComponent.vue";
+import { ZaakZoeker } from "@/features/zaaksysteem";
+import { toast } from "@/stores/toast";
 
-const tabs = { zakenZoeker: "zakenZoeker", personenZoeker: "personenZoeker" };
-const activeTab = ref(tabs.personenZoeker);
+//layout view tabs
+enum TabsContactmoment {
+  klanten = "Klanten",
+  zaken = "Zaken",
+}
+const activeTabContactmoment = ref(TabsContactmoment.klanten);
+
+//zaak tabs
+enum Tabs {
+  zakenZoeker = "Via zaaknummer",
+  personenZoeker = "Via persoon",
+}
+const activeTab = ref(Tabs.personenZoeker);
 const curentBsn = ref<number>();
 
 // er kan direct vanaf de personen tab gezocht worden naar de bijbehorende zaken.
 // we swichen daarvoor naar de zakentab
 const onZakenZoeken = (bsn: number) => {
   curentBsn.value = bsn;
-  activeTab.value = tabs.zakenZoeker;
+  activeTab.value = Tabs.zakenZoeker;
 };
+
+//klant functies
+const showKlantenSearch = ref(true);
+const contactmomentStore = useContactmomentStore();
+
+const klantGevonden = (klant: Klant) => {
+  showKlantenSearch.value = false;
+  contactmomentStore.setKlant(klant);
+  toast({ text: "Klant gekoppeld aan contactmoment" });
```

Comment:
de toast kan eruit wat mij betreft. gaat nu ook af als je dezelfde opnieuw selecteert en is niet consistent met zaken toevoegen aan contactmoment